services/web-server/src/entities/AccessToken.js

```diff
@@ -0,0 +1,37 @@
+const assert = require('assert');
+const Entity = require('azure-entities');
+
+const AccessToken = Entity.configure({
```

Comment:
Has access to these tables been added to `infrastructure/tooling/static-clients.yml`? We should also make sure to have a changelog stating that clients need updated in deployments for cloudops